docker/wiremock/stubs/__files/correctPatientNoDocsStructuredRecordResponse.json

```diff
@@ -243,7 +244,7 @@
                         {
                             "system": "http://snomed.info/sct",
                             "code": "886921000000105",
-                            "display": "Active Allergies"
+                            "display": "Allergies and adverse reactions"
```

Comment:
Just curious, why this change?

Reply:
looks like it had the wrong display in GPC and has been fixed at some point https://termbrowser.nhs.uk/?perspective=full&conceptId1=886921000000105&edition=uk-edition&release=v20210317&server=https://termbrowser.nhs.uk/sct-browser-api/snomed&langRefset=999001261000000100,999000691000001104

Reply:
Came from the public demonstrator update. @mattd-kainos do you think this is an issue?

Reply:
Not an issue, that is the correct display for this snomed ct